### PR TITLE
[12.0][FIX] maintenance_timesheet: python literal text not translatable

### PR DESCRIPTION
**Pull request by dalonsod**
A text defined as variable was wrongly defined in order to be translatable. With this PR the issue is solved

### CODE REVIEW

**Review by astirpe — APPROVED** [review]

**Review by lmiguens-solvos — APPROVED** [review]
LGTM 👍

**Review by jarroyomorales — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I'm not sure this will work, as the translation is performed at the Odoo start, and thus the language taken will be the initial one, not the desired to be used according user language. The previous form has the problem of not being recognized by the POT extract tool, but works if you inject manually the translation. So, the solution is to not use the string literal at all and put the sentence directly on the code.

**Comment by pedrobaeza** [comment]
Aborting the merge!

**Comment by astirpe** [comment]
OK thanks

**Comment by astirpe** [comment]
@dalonsod I just tried on Runbot and it seems that the translation doesn't work, as Pedro said in https://github.com/OCA/maintenance/pull/38#issuecomment-552785927.

Can you put the sentence directly in the code?

**Comment by astirpe** [comment]
@pedrobaeza is it ok to merge it?

File: maintenance_timesheet/tests/test_maintenance_timesheet.py [hunk 1]
```diff
@@ -55,18 +57,18 @@ def test_check_request_done(self):
                 'user_id': self.env.ref('base.user_admin').id,
                 'date': fields.Date.today(),
                 'unit_amount': 2})]
-        self.assertTrue(hr_timesheet.BAD_TIMESHEET_REQUEST_DONE_MSG
+        self.assertTrue(BAD_TIMESHEET_REQUEST_DONE_MSG
```
Comment (pedrobaeza) [inline maintenance_timesheet/tests/test_maintenance_timesheet.py]: This is going to fail if the test user language is not English. I recommend you to not check this. It's enough to check for the rise of the exception.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**